pkg/machine/e2e/init_test.go

```diff
@@ -333,4 +335,38 @@ var _ = Describe("podman machine init", func() {
 			Expect(err).To(HaveOccurred())
 		}
 	})
+
+	It("init should not should not overwrite existing HyperV vms", func() {
+		skipIfNotVmtype(define.HyperVVirt, "HyperV test only")
+		name := randomString()
+		vhdxPath := filepath.Join(testDir, fmt.Sprintf("name.vhdx"))
```

Comment:
Nit: Could we remove `fmt.Sprintf` , not sure if it is needed, can't see any formatted string.
```suggestion
		vhdxPath := filepath.Join(testDir, "name.vhdx")
```